### PR TITLE
update

### PR DESCRIPTION
**Pull request by farukpinar**
Updated bracket type selection.

### CODE REVIEW

File: src/messages/tr-TR/user.php [hunk 1]
```diff
@@ -58,5 +58,5 @@
     'Username Column Name' => 'Kullanıcı Adı Sütunu Adı',
     'Email Column Name' => 'E-posta Sütunu Adı',
     'Password Column Name' => 'Şifre Sütunu Adı',
-    'The Signup Page feature must be activated in the Settings.' => 'Settings içerisinden Signup Page özelliğinin aktif edilmesi gerekiyor.'
```
Comment (sinanbekar) [inline src/messages/tr-TR/user.php:61]: .

File: src/views/backend/import/index.php [hunk 2]
```diff
@@ -28,9 +28,8 @@
 
 <?= $form->field($model, 'email')->textInput(['maxlength' => true]) ?>
 <?= $form->field($model, 'password')->textInput(['maxlength' => true]) ?>
-<?= $form->field($model, 'group')->dropDownList(Group::getGroups(), ['prompt' => Module::t('Not Selected')]) ?>
+<?= $form->field($model, 'group')->dropDownList(\portalium\user\models\Group::getGroups(), ['prompt' => Module::t('Not Selected')]) ?>
```
Comment (sinanbekar) [inline src/views/backend/import/index.php:31]: ```Group::getGroups()```

**Review by sinanbekar — CHANGES_REQUESTED** [review]
Addition to all, ```&& $attribute != "seperator"``` should be removed.
https://github.com/farukpinar/portalium-user/blob/82fe5a6ca7fe4af2005e2bea72e81025e5c81d7e/src/controllers/backend/ImportController.php#L67

**Comment by diginova** [comment]
Please update PR according to requesting changes.

**Pull request closed without merging**